bin/resources/GameIndex.yaml

```diff
@@ -3469,6 +3469,9 @@ SCES-52033:
         // Cop2 problems.
         patch=0,EE,003953F8,word,48438000
         patch=0,EE,003735FC,word,4B06521B
+        author=YukiXXL
+        //Speed Correction (25 FPS)
```

Comment:
can you put a space after the // to match the formatting of all other patches? Thanks.
Then we'll be GTG

Reply:
ok i changed it now, but please check if i actually did it correctly

Reply:
That's fine, but it was my lesser worry about the PR. If you could correct the other thing and make 100% sure the patch works as intended, if it's working now, I fear it might be placebo.